oqamaint/cli.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import os.path as opa
+import sys
+import simplejson as json
+
+import ReviewBot
+from openqabot import OpenQABot
+from suse import SUSEUpdate
+from opensuse import openSUSEUpdate
+from logger import ColorFormatter
+
+from openqa_client.client import OpenQA_Client
+import osc
```

Comment:
Perhaps alphabetize or something consistent?

Reply:
make sense, now is only copy paste from original `openqa-maintenance.py`